### PR TITLE
[CVP-2211] Update cluster pool sizes for cvp-ocp-4.x clusters

### PR DESCRIPTION
**Pull request by samvarankashyap**
Update cluster pool sizes for all CVP-OCP 4.x cluster versions

### CODE REVIEW

**Comment by jsztuka** [comment]
Just out of curiosity, how did we came up with those numbers?

**Review by lslebodn — APPROVED** [review]
LGTM

**Review by jsztuka — APPROVED** [review]
LGTM

File: clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-west-1_clusterpool.yaml [hunk 1]
```diff
@@ -24,5 +24,5 @@ spec:
       region: us-west-1
   pullSecretRef:
     name: pull-secret
-  size: 3
-  maxSize: 10
+  size: 2
```
Comment (sonam1412) [inline clusters/hive/pools/cvp/cvp-ocp-4-6-amd64-aws-us-west-1_clusterpool.yaml:27]: I think the size should be changed to 3 and maxSize: 10 as discussed on the PPT

**Pull request merged**